internal/compiler-interface/src/main/contraband/incremental.json

```diff
@@ -103,13 +103,6 @@
           ]
         },
         {
-          "name": "nameHashing",
```

Comment:
Not using them is making your project's pre-release clients' life hell.

I don't follow all changes that happen in Zinc. But a few times now I've dealt with breaking changes with absolute no guidance.

Maybe we shouldn't update sbt/sbt any more until Zinc can start being more cooperative.